reference/orders.v2.oas2.yml

```diff
@@ -2870,11 +2870,15 @@ components:
                 example: www
                 type: string
               external_source:
-                description: 'For orders submitted or modified via the API, using a PUT or POST operation, you can optionally pass in a value identifying the system used to generate the order. For example: `POS`. Otherwise, the value will be null.'
+                description: |-
+                  The value used to identify an external system that generated the order and submitted it to BigCommerce via the Orders API.
+                  * When supplying the value, we recommend combining the type of system and vendor, e.g. ERP (Acumatica) or POS (Square).
```

Comment:
* When supplying the value, we recommend combining the type of system and vendor, e.g., ERP (Acumatica) or POS (Square).